rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/ui/UnityImportantActions.kt

```diff
@@ -13,6 +15,20 @@ class UnityImportantActions : DefaultActionGroup(), DumbAware {
             return
         }
 
+        e.presentation.isVisible = true
+        e.presentation.icon = UnityIcons.Actions.UnityActionsGroup
+    }
+}
+
+class UnityDllImportantActions : DefaultActionGroup(), DumbAware {
```

Comment:
Why add this as a new action? Why not just change the `update` of `UnityImportantActions` to handle dll projects as well?

Reply:
It has another set of actions inside popup. Also I don't know good API to hide actions and separator which are declared in xml

Reply:
Do the other actions still work in a dll project? If the solution lives inside a Unity project structure, we can still get a connection, so we should be able to show log items, start Unity and install the plugin.

Reply:
Other actions don't work in Dll project. But when dll project lives inside Unity project structure, we will show the basic menu and hide reduced dll menu

Reply:
OK, cool. Thanks!